packages/client/src/components/app/dynamic-filter/DynamicFilter.svelte

```diff
@@ -69,7 +69,10 @@
         }
       })
     }
+
     return Object.values(clonedSchema || {})
+      .filter(field => !Constants.BannedSearchTypes.includes(field.type))
+      .concat([{ name: "_id", type: "string" }])
```

Comment:
`_id` only makes sense for DS+, so we might need to try and be smart here and check if it's a DS+ or not, since currently this is adding it for any data source.

Reply:
That's totally correct. Nice spot 🎉